app/javascript/react/containers/Show.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import ShowLocationTile from '../components/ShowTile'
+import ShowLocationTile from '../components/ShowLocationTile'
 
 class Show extends React.Component {
   constructor(props) {
```

Comment:
There's a fetch that wants to update chosenLocation in state, but that's not here.